docs/snippets/angular/component-story-custom-args-icons.ts.mdx

```diff
@@ -0,0 +1,37 @@
+```ts
+// YourComponent.stories.ts
```

Comment:
`// Icon.stories.ts ` 🤷‍♂️  ?

Reply:
I'm good with this and i've adjusted the code on my end i'll commit the changes shortly.